ui-tests/package.json

```
@@ -0,0 +1,19 @@
+{
+  "name": "jupyterlab-ui-tests",
```

Maybe we can follow the existing pattern and use the `@jupyterlab` scope for naming this package, even though it's private?

```suggestion
  "name": "@jupyterlab/ui-tests",
```